src/reducers/ticketReducers.js

```diff
@@ -45,6 +45,28 @@ const addIncidentsToState = (state, incidents) => {
   return newState
 }
 
+const removeTicketFromState = (state, ticketId) => {
+  let newState = { ...state }
+  newState[ticketId] = null
+  return newState
+}
+
+const removeAllTicketsFromState = (state) => {
+  let newState = { ...state }
+  Object.keys(state).map(ticketId => { newState[ticketId] = null })
```

Comment:
why not just return an empty array []?

Reply:
The state is an object rather than an array, and redux-persist does not persist changes that involve the removal of a key, so returning an empty object would cause redux-persist to re-add all the tickets removed the next time the browser is launched.

Reply:
got it